source/expressions.tex

```diff
@@ -909,7 +909,7 @@
 Otherwise, the behavior is undefined.
 
 \pnum
-The conversions allowed as floating-point promotions are excluded from
+The conversion performed by floating-point promotion is excluded from
```

Comment:
Again, we say "can", so "allowed" seem fine.

Going plural -> singular is, of course, good.